lazy_dataset/core.py

```diff
@@ -3172,8 +3187,15 @@ def __init__(self, input_dataset):
 
     def __repr__(self):
         r = repr(self.input_dataset)
+
+        hits = f'hits = {self.hit_count[0]}'
+        if self.hit_count[1]:
+            # Better alternative for the name "failed"?
```

Comment:
I don't like the name failed.
It describes what the code does, but not what it commonly means.
How about calling it `filtered`? Or something in that direction?

Reply:
How about `"raised an exception"`? These exceptions are not always "filtered". However, the only case in which it is not filtered is when the dataset crashes.